subsys/net/lib/aws_fota/Kconfig

```diff
@@ -21,7 +21,11 @@ config AWS_FOTA_HOSTNAME_MAX_LEN
 
 config AWS_FOTA_FILE_PATH_MAX_LEN
 	int "File path buffer size"
-	default 255
+	default 1246
```

Comment:
Is this default so finetuned or is it just some arbitrary value?

Reply:
Parts of it is fine tuned since they use have hashes for the signature but you are right I should have added some more space for the `file_path`+`file_name` also I should document it in the help text.